remarshal/requirements.txt

```diff
@@ -2,4 +2,4 @@ PyYAML==5.4
 python-dateutil==2.8.0
 pytoml==0.1.20
 remarshal==0.10.0
-six==1.12.0
+six==1.16.0
```

Comment:
Since it's using the system Python install, is this modifying Python packages outside of a venv?

Reply:
No virtualenv is in use.  The binary was generated by invoking bazel to rebuild the `remarshal` par file.  It uses the [subpar](https://github.com/google/subpar) tooling to do this.

Theoretically, the PAR file's inputs are managed by bazel itself using whatever python toolchain is configured, so it theoretically should only contain what is requested (and any non-system dependencies).

The `.par` file generated is should be a zip archive; you can extract it if you'd like to confirm further.

Reply:
Sounds good.  I mainly wanted to understand how it was working for my own edification 👍🏼.

Reply:
Aye.  Yes, that should not be happening.  Thanks for checking!